src/Components/WebAssembly/WebAssembly/src/Http/RequestCache.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace System.Net.Http
```

Comment:
@rynowak , @terrajobst, do you have any thoughts on us placing these types in this namespace? These are APIs specifically meant to program against the WebAssembly HTTP Message handler, so there's precedent to doing this: https://github.com/dotnet/runtime/blob/master/src/libraries/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs#L21-L34. 

It makes it much more discoverable, compared to putting it in a Blazor specific namespace such as `Microsoft.AspNetCore.Components.WebAssembly` or `Microsoft.AspNetCore.Components.WebAssembly.Web`. 

Ideally, this would be exposed as part of the runtime contract. Using these namespaces would make it much more reasonable to type forward to if we do choose to move them as part of 5.0.

Reply:
I think if I were @terrajobst - I'd be very concerned about us parking a generic name like `RequestCache` in a package that a lot of people are going to be using.

Reply:
It would be reasonably discoverable if in `Microsoft.AspNetCore.Components.Web`, wouldn't it? That namespace is imported by default in `.razor` files in Blazor projects. And in `.cs` files in any project, the IDE will help with completion and namespace imports.

Reply:
@rynowak isn't this very close to the WinHttpHandler APIs I linked? Is the concern that it's part of the Components.WebAssembly package rather than a more specific package - https://www.nuget.org/packages/System.Net.Http.WinHttpHandler?

@SteveSandersonMS the namespace doesn't appear as part of `Program.cs`. So it's less discoverable if you were trying to configure the behavior globally.

Reply:
@rynowak is following up on this with @terrajobst offline. I'll park this in the `Microsoft.AspNetCore.Components.WebAssembly.Web` namespace for the time being.

Reply:
@pranavkm I'm mainly concerned about usability in `.razor` files, since in `.cs` the IDE will provide auto-insertion of namespaces for extension methods. I don't think we have `Microsoft.AspNetCore.Components.WebAssembly.Web` in `_Imports.razor` by default.